server/views/pages/goal/add-step/index.njk

```diff
@@ -54,9 +54,6 @@
             errorMessage: errors | findError('title')
           }) }}
 
-          {% set targetDateError = errors | findError('targetDate') %}
-          {% set targetDateErrorClass = " govuk-input--error" if targetDateError %}
```

Comment:
This was a hangover from when we were using 3 separate fields to represent a date. My bad for not updating this properly (partly because I knew even less then that I do now!).